unstructured/ingest/connector/fsspec.py

```diff
@@ -69,10 +69,10 @@ def _tmp_download_file(self):
 
     @property
     def _output_filename(self):
-        return (
-            Path(self.processor_config.output_dir)
-            / f"{self.remote_file_path.replace(f'{self.connector_config.dir_path}/', '')}.json"
+        filename = (
+            f"{self.remote_file_path.replace(self.connector_config.path_without_protocol, '')}.json"
```

Comment:
Are you sure this is going to work for nested folders? Or maybe we not nesting anymore? (It's been a long time since I've thought about the where the partitioned files go.) 

As I test S3 with this, there is no `path_without_protocol` even though there is a nested folder path. So if 2 files have the same name or are in different folders maybe this will be an issue.

Reply:
When I tested it locally it maintained the nested path. Can you post exactly what you ran where the sub folders weren’t in the output path?

Reply:
Ok. Let me look into it a little deeper. I was just comparing the result of the new _output_filename to the old _output_filename for S3. I'll have to run the full test to check.

Just curious, how far down the pipeline do nested paths go?

Reply:
I got this with Box:
`INFO     Copying /Users/davidpotter/Documents/Unstructured/sessions/unstructured/test_unstructured_ingest/workdir/box/partitioned/f354b481320e7405c87cbcd32bde8fce.json -> /handbook-1p.docx.json`
It tried to write the file to my root directory.

Reply:
You've probably seen how the Path module barfs when it gets "/filename.txt" after the ` / ` I wish it was smarter.

Reply:
let's add a test case for this ^^

Reply:
but this is working in CI and the box test has nested folders?

Reply:
@ryannikolaidis Where did it pass CI? Last CI I see is with `update unit tests` `a43e58c` and it did't get to run box.sh (which is probably in 3.10. (maybe it ran CI since then and I don't see it. I would like to learn where to find it if thats the case)